aws-ts-apigateway-lambda-serverless/index.ts

```diff
@@ -1,38 +1,63 @@
 // Copyright 2016-2025, Pulumi Corporation.  All rights reserved.
 
-import * as awsx from "@pulumi/awsx";
+import * as aws from "@pulumi/aws";
+import * as apigateway from "@pulumi/aws-apigateway";
+import * as pulumi from "@pulumi/pulumi";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import handler from "./handler";
 
 /**
- * api-gatewayx https://www.pulumi.com/docs/guides/crosswalk/aws/api-gateway/
+ * api-gateway https://www.pulumi.com/docs/guides/crosswalk/aws/api-gateway/
  */
 
+// Create Lambda functions for our API
+const handlerFunction = new aws.lambda.CallbackFunction("get-handler", {
+  callback: handler,
+  runtime: aws.lambda.Runtime.NodeJS18X,
+});
+
+const postHandlerFunction = new aws.lambda.CallbackFunction("post-handler", {
+  callback: async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    console.log("Inline event handler");
+    console.log(event);
+    return {
+      statusCode: 200,
+      body: JSON.stringify({ message: "POST successful" }),
+    };
+  },
+  runtime: aws.lambda.Runtime.NodeJS18X,
+});
+
+const deleteHandlerFunction = new aws.lambda.CallbackFunction("delete-handler", {
+  callback: async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    console.log("Inline delete event handler");
```

Comment:
this is kind of a lie now. I think the point had previously been to show how you could write a handler inline vs pointing to a function. Now these are all `CallbackFunction` instances.